apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/monetary/model/Asset.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.monetary.model;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import com.apollocurrency.aplwallet.apl.core.app.Transaction;
+import com.apollocurrency.aplwallet.apl.core.db.DbKey;
+import com.apollocurrency.aplwallet.apl.core.db.model.VersionedDerivedEntity;
+import com.apollocurrency.aplwallet.apl.core.monetary.dao.AssetTable;
+import com.apollocurrency.aplwallet.apl.core.transaction.messages.ColoredCoinsAssetIssuance;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+public class Asset extends VersionedDerivedEntity {
+
+    private long assetId;
```

Comment:
Looks like a duplication of class name, assuming that method getAssetId() is typically calling from `asset` object, the following construction `asset.getAssetId()` is redundant

Reply:
Renamed to `id / getId();`